.github/workflows/build.yml

```diff
@@ -87,18 +87,15 @@ jobs:
           pwd
           mkdir build
           cd build
-          cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE ..
+          cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DK2_ENABLE_NVTX=OFF ..
```

Comment:
Does this turn it off?  I don't think nvtx adds much cost so I'd rather have it on at the moment while debugging is our main priority.

Reply:
Yes, it disables NVTX for building wheel packages. I thought wheel files do not need NVTX.
Will revert it.